bamboo/set-bamboo-env-variables.sh

```diff
@@ -101,6 +101,19 @@ if [[ $bamboo_NGAP_ENV = "SIT" ]]; then
   DEPLOYMENT=$bamboo_SIT_DEPLOYMENT
 fi
 
+## Override KES_DEPLOYMENT
+## Delete this when CI is switched to terraform for all of Core Team
+## (after v1.15, i.e. last non-Terraform release)
+if [[ $KES_DEPLOYMENT != true ]]; then
```

Comment:
I'm presuming we're setting KES_DEPLOYMENT to true in the build configuration?

Reply:
We can do that when it merges to master, yes.